lib/Components/logout_alert.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flood_mobile/Constants/app_color.dart';
+import 'package:flutter/material.dart';
+
+class LogOutAlert extends StatelessWidget {
+  const LogOutAlert({Key? key, @required this.logoutOnClick})
+      : assert(logoutOnClick != null),
+        super(key: key);
+  final VoidCallback? logoutOnClick;
+
+  @override
+  Widget build(BuildContext context) {
+    var deviceHeight = MediaQuery.of(context).size.height;
```

Comment:
As per convention please name the deviceHeight variable as hp and if you ever use the deviceWidth, name it as wp.

Reply:
Noted